addons/sale_management/models/sale_order.py

```diff
@@ -152,7 +152,7 @@ def action_confirm(self):
         res = super(SaleOrder, self).action_confirm()
         for order in self:
             if order.sale_order_template_id and order.sale_order_template_id.mail_template_id:
```

Comment:
Does the portal/public user even have access to `order.sale_order_template_id.mail_template_id` ?

Reply:
?